src/camera_node.cpp

```diff
@@ -8,13 +9,16 @@ int main(int argc, char * argv[])
     rclcpp::init(argc, argv);
 
     std::cout << "DepthAI Camera Node." << std::endl;
+    rclcpp::executors::MultiThreadedExecutor exec;
+    rclcpp::NodeOptions options;
     auto cameraNode = std::make_shared<DepthAICamera>();
+    exec.add_node(cameraNode);
 #ifdef CONTINUE_TRYING_TO_CONNECT
     while(rclcpp::ok())
```

Comment:
The try...catch at least looks fishy, does this while even do anything reasonable?

Reply:
Multithreadedexecuter was added to camera_node. The other while, try.. catch does not really do anything, as left as the code in main branch. All the faults are handled in the main nodes anyways, so there is no need for these. I will cancel this PR because there have been more changes regarding the build checks and tests. I will re-open a cleaner PR.